### PR TITLE
Package cv-satysfi 0.2.0

### PR DESCRIPTION
**Pull request by MasWag**
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-class-cv-doc.0.2.0 satysfi-class-cv.0.2.0
- [x] Add to snapshot `snapshot-stable-0-0-10` :: satysfi-class-cv-doc.0.2.0 satysfi-class-cv.0.2.0
- [x] Add to snapshot `snapshot-stable-0-0-11` :: satysfi-class-cv-doc.0.2.0 satysfi-class-cv.0.2.0
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-6`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-6--1` :: satysfi-class-cv-doc.0.2.0 satysfi-class-cv.0.2.0
- [x] Add to snapshot `snapshot-stable-0-0-7` :: satysfi-class-cv-doc.0.2.0 satysfi-class-cv.0.2.0
- [x] Add to snapshot `snapshot-stable-0-0-8` :: satysfi-class-cv-doc.0.2.0 satysfi-class-cv.0.2.0

### CODE REVIEW

File: packages/satysfi-class-cv-doc/satysfi-class-cv-doc.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+authors: [
+    "Masaki Waga <masakiwaga@gmail.com>"
+]
+license: "MIT"
+homepage: "https://github.com/MasWag/cv-satysfi"
+bug-reports: "https://github.com/MasWag/cv-satysfi/issues"
+dev-repo: "git+https://github.com/MasWag/cv-satysfi.git"
+
+depends: [
+  "satysfi" {>= "0.0.3" & < "0.0.9"}
```
Comment (na4zagin3) [inline packages/satysfi-class-cv-doc/satysfi-class-cv-doc.0.2.0/opam:15]:
```suggestion
  "satysfi" {>= "0.0.3" & < "0.1"}
```

**Review by na4zagin3 — COMMENTED** [review]
Thank you!  Do you mind updating the upper bound of `satysfi`?

**Comment by MasWag** [comment]
Thanks for your comment! I have updated the dependencies of class-cv-doc.

**Review by na4zagin3 — APPROVED** [review]

**Comment by na4zagin3** [comment]
Thank you!

**Pull request merged**